nvflare/job_config/fed_job_config.py

```diff
@@ -0,0 +1,321 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import builtins
+import inspect
+import json
+import os
+import shutil
+from enum import Enum
+from typing import Dict
+
+from nvflare import SimulatorRunner
+from nvflare.job_config.fed_app_config import FedAppConfig
+from nvflare.private.fed.app.fl_conf import FL_PACKAGES
+
+CONFIG = "config"
+CUSTOM = "custom"
+FED_SERVER_JSON = "config_fed_server.json"
+FED_CLIENT_JSON = "config_fed_client.json"
```

Comment:
While JSON is OK, we also have .conf file, which one do we prefer @chesterxgchen

Reply:
We could add other format of config file support as options as enhancement in the future.